examples/broken/correlation_alignment_example.py

```diff
@@ -16,6 +18,71 @@
 from collections import OrderedDict
 from numpy.random import seed
 
+
+# {{{ Define the frequency mask function and the ph cyc mask
+def frq_mask(s, signal_pathway, direct="t2", indirect="repeats"):
+    """Generates a mask that is nonzero along frequencies only over the
+    bandwidth of the signal
+        Parameteres
+        ===========
+        s: nddata
+            data that the mask is applied to
+        signal_pathway: dict
+            This is the default pathway that will be nonzero since this is
+            where your signal is.
+        direct: str
+            Direct dimension
+        indirect: str
+            The indirect dimension (does not include phase cycling dimensions)
+
+    Returns
+        =======
+        s: nddata
+            copy of data with the mask applied
+    """
+    # Just for consistency we will start with the frequency domain in the phase
+    # cycling dimension
+    assert s.get_ft_prop(direct), "You must be in the frequency domain!"
+    for phnames in signal_pathway.keys():
+        assert not s.get_ft_prop(phnames), (
+            str(phnames) + "must NOT be in coherence domain!"
+        )
+    signal_keys = list(signal_pathway)
+    signal_values = list(signal_pathway.values())
+    # we want to leave the original s unchanged and return a copy
+    for_mask = s.C
+    for_mask.ft(list(signal_pathway))
+    # {{{ The following is all pulled directly from the correlation alignment
+    #     function
+    # {{{ find center frequency
+    for j in range(len(signal_keys)):
+        signal = for_mask[signal_keys[j], signal_values[j]].C
+    nu_center = signal.mean(indirect).C.argmax(direct)
+    # }}}
+    # {{{ Make mask using the center frequency and sigma (whose estimate here
+    #     is 20)
+    frq_mask = np.exp(
+        -((for_mask.fromaxis(direct) - nu_center) ** 2) / (2 * 20.0**2)
+    )
+    # }}}
+    for_mask.ift(list(signal_pathway))
+    # }}}
+    return for_mask * frq_mask
```

Comment:
OK, I've thought about this a bit, and the square root of the frequency domain mask should multiply s_j,k (the one that does not have the extra dimension, and that does not have the complex conjugate)

Reply:
I think we've resolved this